### DISCUSSION TITLE
Release transaction enhancements

### DISCUSSION THREAD

**Discussion by jacobyavis** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#243] [General]
Missing fields:
release_date
resulting_security_ids

Minor: I think we need to improve the descriptions/documentation on the release fields since there are a lot of them.

Is there a reason we have so many extra fields for releases as compared to exercises? Usually the same fields should be available to exercises as well, though what is there now would suffice. The releases just seem to have extra fields.

**Comment by JSv4** [comment]
@EricVogl-carta, you have any thoughts on this?

**Comment by EricVogl-carta** [comment]
For reference, the current schema for a release is https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/blob/v1.0.0-beta/schema/objects/transactions/release/PlanSecurityRelease.schema.json

The current fields are:
```
"id": {},
    "comments": {},
    "security_id": {},
    "date": {},
    "settlement_date": {},
    "release_price": {},
    "net_quantity": {},
    "method": {},
    "stock_swap": {},
    "cash_paid": {},
    "quantity_sold": {},
    "sale_price_per_unit": {},
    "withheld_quantity": {}
```

I'm assuming `date` would be the release date, if so I don't think we need this additional field. If it is something else (date that the release was run versus the effective release date?), then we should have an explicit field for the release date.

The resulting security IDs would also be useful for any releases that result in settled shares (most private RSU releases would have this).

**Reply by pjohnmeyer** [reply]
You are correct that `date` is the release date. We can add `resulting_security_ids`.
